src/includes/class-alma-wc-payment-gateway.php

```diff
@@ -839,14 +849,7 @@ public function process_payment( $order_id ) {
 				Alma_WC_Model_Payment::get_payment_payload_from_order( $order_id, $fee_plan_definition )
 			);
 		} catch ( RequestError $e ) {
-			$this->logger->log_stack_trace(
-				'Error while creating payment.',
-				$e,
-				array(
-					'OrderId'           => $order_id,
-					'FeePlanDefinition' => $fee_plan_definition,
-				)
-			);
+			$this->logger->log_stack_trace( 'Error while creating payment: ', $e );
```

Comment:
why did you removed this informations, order_id is one of the context information we want in the error context to help to debug if necessary ...